publication-commons/src/main/java/no/unit/nva/publication/model/business/ImportCandidate.java

```diff
@@ -26,7 +26,8 @@
 public class ImportCandidate extends Publication implements JsonSerializable {
 
     public static final String TYPE = "ImportCandidate";
-    @JsonProperty("importStatus")
+    public static final String IMPORT_STATUS = "importStatus";
```

Comment:
when IMPORT_STATUS == "importStatus" == fieldName
You are just adding text without any purpose, i.e. your code gets longer without any benefits.

It is a pattern that is repeated all over, so I will not ask you to change it, but maybe you should ask why you are doing this?